templates/seL4VMParameters.template.h

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright 2023, DornerWorks
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
```

Comment:
Please add an include guard. Seem we don't need any special headers here as long as we just use plain C integers.

```suggestion
#pragma once
```

Reply:
Will do.